pandas/tests/scalar/test_period.py

```diff
@@ -1038,6 +1038,24 @@ def test_add_raises(self):
         with tm.assert_raises_regex(TypeError, msg):
             dt1 + dt2
 
+    boxes = [lambda x: x, lambda x: pd.Series([x]), lambda x: pd.Index([x])]
+
+    @pytest.mark.parametrize('lbox', boxes)
+    @pytest.mark.parametrize('rbox', boxes)
+    def test_add_timestamp_raises(self, rbox, lbox):
+        # GH # 17983
+        ts = pd.Timestamp('2017')
+        per = pd.Period('2017', freq='M')
+
+        with pytest.raises(TypeError):
+            lbox(ts) + rbox(per)
+
+        with pytest.raises(TypeError):
+            lbox(per) + rbox(ts)
```

Comment:
can you add a similar test with Series of periods & Timestamp; I think these are now in pandas/tests/series/test_timestamp.py IIRC (maybe make a test_period.py  in series ok too).

Reply:
woops, just pushed without addressing this.  Do you mean a Series that mixed, contains both Periods and Timestamps?